sdk/instrumentation/opentelemetry-instrumentation-azure-sdk/src/instrumentation.ts

```diff
@@ -61,7 +65,7 @@ class AzureSDKInstrumentation extends InstrumentationBase {
  * ```ts
  * const openTelemetryInstrumentation = require("@opentelemetry/instrumentation");
  * openTelemetryInstrumentation.registerInstrumentations({
- *   instrumentations: [createAzureInstrumentation()],
+ *   instrumentations: [createAzureSDKInstrumentation()],
```

Comment:
Suuper nitpicky, but usually even when an acronym is written in a function name, we typically camel case it.

`createAzureSdkInstrumentation`

Reply:
Thanks! Fixed